src/ui/Facet/FacetValuesOrder.ts

```diff
@@ -42,8 +42,8 @@ export class FacetValuesOrder {
   }
 
   private reorderValuesWithCustomCaption(facetValues: FacetValue[]) {
-    let sorted = _.sortBy(facetValues, (facetValue: FacetValue) => {
-      return this.facet.getValueCaption(facetValue).toLowerCase();
+    let sorted = facetValues.sort((firstValue, secondValue) => {
+      return this.facet.getValueCaption(firstValue).toLowerCase().localeCompare(this.facet.getValueCaption(secondValue).toLowerCase(), String['locale']);
```

Comment:
You can use `{sensitivity:'base'}` as option to `localeCompare()`.

For example: 
`this.facet.getValueCaption(firstValue).localeCompare(this.facet.getValueCaption(secondValue), String['locale'], {sensitivity:'base'});`

Lowest comformance is IE 11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare